content/machine-config/_index.md

```diff
@@ -133,3 +133,52 @@ spec:
         path: /etc/sysctl.d/99-elasticsearch.conf
 EOF
 ```
+
+# Modify MTU in a second interface in workers
+
+Sometimes a storage network interface is attached to nodes in order to reach an external storage. Probably you could need to modify the MTU in those interfaces. 
+
+You can do that adding an an script in dispatcher.d for NetworkManager service in the path `/etc/NetworkManager/dispatcher.d/`. But if SELinux is enabled in your installation you could have errors when the daemon execute that script. For that you should add a new one-shot systemd service for modify the context. 
+
+In this example the MTU of the interface `ens4` will change to `9000` (jumbo)
```

Comment:
In this example the MTU of the `ens4` interface will change to `9000` to enable jumboframes:

Reply:
Change done, thanks